docs/container.md

```diff
@@ -0,0 +1,120 @@
+
+
+- **pulpcore-resource-manager** - a different type of pulp-worker that plays a coordinating role for the tasking system. All asynchronous work flows through this service. Only a single entity does work, but other instances can be run as hot spares that will take over if the active one fails.
+
+
+Currently built with the following plugins:
+
+* [pulp_ansible](https://docs.pulpproject.org/pulp_ansible/)
+* [pulp-certguard](https://docs.pulpproject.org/pulp_certguard/)
+* [pulp_container](https://docs.pulpproject.org/pulp_container/)
+* [pulp_deb](https://docs.pulpproject.org/pulp_deb/)
+* [pulp_file](https://docs.pulpproject.org/pulp_file/)
+* [pulp_python](https://docs.pulpproject.org/pulp_python/)
+* [pulp_rpm](https://docs.pulpproject.org/pulp_rpm/)
+
+### Tags
+
+* `latest`: Nightly built, with master/main branches of each plugin.
+* `stable`: Built on push, with latest released version of each plugin.
+* `3.y.z`:  Pulpcore 3.y.z version and its compatible plugins.
+
+[https://quay.io/repository/pulp/pulp?tab=tags](https://quay.io/repository/pulp/pulp?tab=tags)
+
+
+## Pulp Web
+
+An Nginx imaged based on [centos/nginx-116-centos7](https://hub.docker.com/r/centos/nginx-116-centos7),
+with pulpcore and plugins specific configuration.
+
+### Tags
+
+* `latest`: Nightly built, with master/main branches of [pulpcore](https://github.com/pulp/pulpcore) and its plugins.
+* `stable`: Built on push, with latest released version of each plugin.
+* `3.y.z`:  Pulpcore 3.y.z version and its compatible plugins.
+
+[https://quay.io/repository/pulp/pulp-web?tab=tags](https://quay.io/repository/pulp/pulp-web?tab=tags)
+
+
+## Galaxy
+
+An all-in-one [galaxy](https://github.com/ansible/galaxy_ng) image that can assume each of the following types of service:
+
+- **pulpcore-api** - serves the Galaxy (v3) API. The number of instances of this service should be scaled as demand requires.  `Administrators and users of all of the APIs put demand on this service`.
+
+
+- **pulpcore-content** - serves content to clients. pulpcore-api redirects clients to pulpcore-content to download content. When content is being mirrored from a remote source, this service can download that content and stream it to the client the first time the content is requested. The number of instances of this service should be scaled as demand requires. `Content consumers put demands on this service`.
+
+
+- **pulpcore-worker** - performs syncing, importing of content, and other asynchronous operations that require resource locking. The number of instances of this service should be scaled as demand requires. `Administrators and content importers put demands on this service`.
+
+
+- **pulpcore-resource-manager** - a different type of pulp-worker that plays a coordinating role for the tasking system. All asynchronous work flows through this service. Only a single entity does work, but other instances can be run as hot spares that will take over if the active one fails.
```

Comment:
pulpcore-resource-manager is a different kind of pulp-worker, pulpcore-resource-manager is the one who orchestrates pulp-workers, so we have just one entity

> What is an entity = the pulp-worker?

pulpcore-resource-manager (which is a different type of pulp-worker)

> You can only have one of these and hot spares available if one goes down?

Yes, only one `pulpcore-resource-manager` as it coordinates the pulp-workers

Reply:
Is this somewhat close to where we want to be: 
```suggestion
- **pulpcore-resource-manager** - a different type of pulp-worker that plays a coordinating role for the tasking system. All asynchronous work flows through this service. Because `pulpcore-resource-manager` coordinates pulp-workers, it is only possible to run one at a time, but other instances can be run as hot spares that will take over if the active one fails.
```

Reply:
yes! thank you!

Reply:
Excellent!